docs/device-preparation.md

```diff
@@ -32,6 +32,12 @@ Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Adva
 Make sure the destination `WKWebView` and/or `JSContext` component have [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) property set to `true`.
 Read [Enabling the Inspection of Web Content in Apps](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/) for more details on this property.
 
+#### Chrome v115+ and iOS 16.4+ support Web testing availability on Chrome
+
+Please turn on _Web Inspector_ on iOS device via Chrome app -> _Settings_ -> _Content Settings_ -> _Web Inspector_ -> Turn _Web Inspector_ on -> Kill the Chrome app process
```

Comment:
>  Kill the Chrome app process

Is it the setting name?

Reply:
no, it is via app switcher or app history. I have modified a bit, whats about now?